Sparkle/fr.lproj/Sparkle.strings

```diff
@@ -4,6 +4,8 @@
 
 "%1$@ can't be updated, because it was opened from a read-only or a temporary location. Use Finder to copy %1$@ to the Applications folder, relaunch it from there, and try again." = "%1$@ ne peut pas être mis à jour quand il fonctionne à partir d’un volume en lecture seule, comme une image disque ou un lecteur optique. Déplacez %1$@ dans votre dossier Applications, relancez-le à partir de là et réessayez.";
 
+"Quit %1$@, move it into your Applications folder, relaunch it from there and try again. %2$@ can’t be updated if it’s running from the location it was downloaded to." = "Quittez %1$@ et déplacez-le dans le dossier Applications, relancez-le à partir de ce dossier et essayez à nouveau. Transmit ne peut pas être mis à jour s'il est exécuté depuis le dossier dans lequel il a été téléchargé.";
```

Comment:
Perhaps it should be `%2$@` instead of of "Transmit" in the localized string.

Reply:
Thanks for catching that. I have pushed a fix. Sorry for the inconvenience.